content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -552,16 +560,20 @@ Additionally, the following restrictions are applied to the schema:
   - `writeOnly`,
   - `xml`,
   - `$ref`.
-- The field `uniqueItems` cannot be set to _true_.
-- The field `additionalProperties` cannot be set to _false_.
+- The field `uniqueItems` cannot be set to `true`.
+- The field `additionalProperties` cannot be set to `false`.
 - The field `additionalProperties` is mutually exclusive with `properties`.
 
 These fields can only be set with specific features enabled:
-
-You can also use [Validation Schema Defaulting](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting) to apply default values.
+ 
+- `default`: can be set for `apiextensions.k8s.io/v1` CustomResourceDefinitions.
+   Defaulting is in GA since 1.17 (beta since 1.16 with the `CustomResourceDefaulting`
+   feature gate enabled, which is the case automatically for many clusters for beta
```

Comment:
nit: we'd usually link “feature gate” to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/